torch/csrc/jit/script/compiler.cpp

```diff
@@ -1237,14 +1519,6 @@ struct to_ir {
         auto kind = getNodeKind(tree->kind(), inputs.size());
```

Comment:
Should the UNARY_MINUS stay here? Will this case ever get triggered?

Reply:
Will change

Reply:
Resolved; I moved the other UNARY_MINUS case to this location so that this one is the only one triggered.